doc/source/reshaping.rst

```diff
@@ -402,6 +404,9 @@ It takes a number of arguments
 - ``colnames``: sequence, default None, if passed, must match number of column
   arrays passed
 - ``margins``: boolean, default False, Add row/column margins (subtotals)
+- ``normalize``: boolean, {'all', 'index', 'columns'}, or {0,1}, default False.
+   Normalize by dividing all values by the sum of values.
```

Comment:
Very small detail, but I think there is one space too much at the beginning of this line